includes/export/abstract-wc-csv-exporter.php

```diff
@@ -296,7 +296,7 @@ protected function export_row( $row_data, $key, $buffer ) {
 	 * @return int
 	 */
 	public function get_limit() {
-		return $this->limit;
+		return apply_filters( "woocommerce_product_export_get_limit", $this->limit, $this->export_type);
```

Comment:
Better change to:

```php
return apply_filters( "woocommerce_{$this->export_type}_export_batch_limit", $this->limit, $this );
```

Better filter naming, since this that class may or not be for products, and this fix the coding style issues too.